docs-ref-conceptual/use-azure-cli-successfully-shorthand.md

```diff
@@ -0,0 +1,364 @@
+```
+
+## Partial value shorthand syntax
+
+Shorthand syntax for partial value is composed of two parts joined by an equal (`=`) sign between the index `key` and the `value`.
+Here's an example: `key=value`.
+
+The value can be a simplified string, full value format, JSON, or a JSON file path. Using the JSON example provided, pass properties for the `--contact` parameter following these examples:
+
+Use `Partial Value` for a single key-value pair.
+
+```bash
+az some-command --contact name=Bill
+```
+
+Use `Partial Value` for two key-value pairs. Note the space that separates the two pairs.
+
+```bash
+az some-command --contact age=20 paid=true
+```
+
+Use `Partial Value` for second element.
+
+```bash
+az some-command --contact emails[1]="Bill@outlook.com"
+```
+
+Use `Partial Value` for _details_ property of _address_.
+
+```bash
+az some-command --contact address.details="{line1:'15590 NE 31st St',line2:'Redmond, WA'}"
+```
+
+## Combine full and partial value shorthand syntax
+
+You can combine full and partial value syntax, but always start with `Full Value` followed by `Partial Value`. If you reverse the order, the final data only contains the `Full Value` without properties defined in `Partial Value`.
+
+Use `Full Value` followed by `Partial Value`:
+
+```bash
+az some-command --contact "{name:Bill,age:20,paid:true,emails:[Bill@microsoft.com,Bill@outlook.com]}" motto="One man's bug is another man's lesson."
+```
+
+You can also patch a new element of list property in `Full Value`. For example you can set the second email address by `Partial Value`:
+
+```bash
+az some-command --contact "{name:Bill,age:20,paid:true,emails:[Bill@microsoft.com]}" emails[1]="Bill@outlook.com" motto="One man's bug is another man's lesson."
+```
+
+## Pass JSON in a file
+
+It's also possible to pass a JSON file as a parameter value. This is the recommended approach when working with complex JSON.
+
+```bash
+az some-command --contact address.details=./address_details.JSON
+```
+
+## Single Quotes String
+
+_Single Quotes String_ is used to pass a __string__ value with special characters: `:`, `,`, `{`, `}`, `[`, `]`, `null`, `??`, and space.
+These characters often have other meanings when parsing shorthand syntax. Single quotes tell the parser to treat everything as a string.
+
+## Pass a string value with a space and other special characters
+
+Here's example JSON to pass as a value in the `--contact` parameter:
+
+```JSON
+{
+  "name": "Bill RP",
+  "age": 20,
+  "paid": true,
+  "data": "{a: [1, 2]}"
+}
+```
+
+Use single quotes in `Full Value` format:
+
+```bash
+az some-command --contact "{name:'Bill RP',age:20,paid:true,data:'{a: [1, 2]}'}"
+```
+
+Use single quotes in `Partial Value` format:
+
+```bash
+az some-command --contact name="'Bill RP'" data="'{a: [1, 2]}'"
+```
+
+In the next example, it's also possible to remove single quotes for the `name` key. The parser doesn't distinguish between a `Full Value` expression, `null` value, or the `??` flag.
```

Comment:
@@kairu-ms, Did I rewrite this sentence correctly for your original `It's also possible to remove single quotes for name, because it will not distinguish with Full Value expression, null value or ?? flag. Pass Partial Value format:`?

Reply:
Yes, it's correct